lib/config/folderconfiguration.go

```diff
@@ -164,6 +166,11 @@ func (f *FolderConfiguration) prepare() {
 		f.RescanIntervalS = 0
 	}
 
+	if f.NotifyDelayS <= 0 {
+		f.FsNotifications = false
+		f.NotifyDelayS = 10
```

Comment:
For consistency, I'd call the variables

FsNotify or FsNotifyEnabled
FsNotifyDelay

but I'd like to see what @calmh thinks.